packages/cli/src/node-types.ts

```diff
@@ -60,6 +61,13 @@ export class NodeTypes implements INodeTypes {
 			description: { value: clonedDescription },
 		}) as INodeType;
 		const tool = this.loadNodesAndCredentials.convertNodeToAiTool(clonedNode);
+		if (!tool.execute) {
```

Comment:
Does absence of execute method 100% mean the node is a routing node?

Reply:
yeah. we should have ideally created a separate set of types for these nodes, but we didn't, and now this is the only way to identify such nodes in this part of the codebase.